index.bs

```diff
@@ -558,8 +558,13 @@ Instances of <code>ReadableStream</code> are created with the internal slots des
 </div>
 
 <ol>
-  <li> If <b>this</b>@\[[reader]] is not <b>undefined</b>, return <b>this</b>@\[[reader]]@\[[lockReleased]].
-  <li> Return <b>this</b>@\[[readyPromise]].
+  <li> If <b>this</b>@\[[readableStreamReader]] is not <b>undefined</b>, return the result of transforming
+    <b>this</b>@\[[readableStreamReader]]@\[[lockReleased]] by a fulfillment handler that returns <b>this</b>@\[[readyPromise]].
+  <li> Return the result of transforming <b>this</b>@\[[readyPromise]] by the following fulfillment handler:
+    <ol>
+      <li> If <b>this</b>@\[[readableStreamReader]] is <b>undefined</b>, return <b>undefined</b>.
+      <li> Otherwise, return <b>this</b>@\[[readableStreamReader]]@\[[lockReleased]].
+    </ol>
```

Comment:
Ah, I see. I think this is analogous to the case without any reader involved:

```js
stream.ready.then(() => console.log(stream.state));

// enqueue a chunk
stream.read();

// will log "waiting"
```

This is somewhat of a more general problem. (If it is a problem.) I will open a new bug to discuss it, although it might be fine to leave as-is.